nautobot/extras/forms/forms.py

```diff
@@ -611,12 +611,14 @@ class Meta:
             "tags",
         ]
 
-    def clean(self):
-        super().clean()
+    def save(self, commit=True):
+        instance = super().save(commit=commit)
```

Comment:
If there's a use-case for `commit=False`, the code below needs to account for that and probably not run unless `commit=True`.

Reply:
Ah, I think I'm starting to remember now why we had used transaction.on_commit in the original implementation - I think there are (or maybe were?) some cases where we were saving the GitRepository with commit=false specifically to prevent triggering an automatic resync or dry-run... I can't remember more specifics than that at the moment.